webapp/views/wpt-results.js

```diff
@@ -988,21 +1003,37 @@ class WPTResults extends AmendMetadataMixin(Pluralizer(WPTColors(WPTFlags(PathIn
     return canViewInteropScores && !pathIsATestFile;
   }
 
+  testViewButtonClass(view) {
+    return (view === 'test') ? 'selected' : 'unselected';
+  }
+
   interopButtonClass(view) {
     return (view === 'interop') ? 'selected' : 'unselected';
   }
 
   defaultButtonClass(view) {
-    return (view !== 'interop') ? 'selected' : 'unselected';
+    return (view !== 'interop' && view !== 'test') ? 'selected' : 'unselected';
   }
 
   clickInterop() {
-    if (!this.isDefaultView()) {
+    if (this.isInteropView()) {
       return;
     }
     this.view = 'interop';
   }
 
+  clickTestView() {
```

Comment:
nit: consider declaring a string constant for `interop`, `test` on the top

Reply:
Done!